package.json

```diff
@@ -5,8 +5,13 @@
   "main": "index.js",
   "scripts": {
     "start": "node build/index.js",
+<<<<<<< HEAD
     "pretest": "cross-env NODE_ENV=test npm run migrate:undo && cross-env NODE_ENV=test npm run migrate && cross-env NODE_ENV=test npm run seed",
     "test": "cross-env DEBUG=dev jest --forceExit --verbose --maxWorkers=1 --coverage",
+=======
+    "pretest": "npm run migrate:undo && npm run migrate && npm run seed",
+    "test": "cross-env DEBUG=dev jest --updateSnapshot --forceExit --verbose --maxWorkers=1 --coverage",
+>>>>>>> 3395de6e5a306be417fd8b3be2bbf028d39d27ec
```

Comment:
You didn't handle merge conflict well

Reply:
Oops, I didn't see that file. Will fix that ASAP. Still on my way